recipes/octo-logger-cpp/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
```

Comment:
modern cmake practice is to use lower case for the functions (everything that's highlight blue) 🙏 it just styling but this would be the only one to break convention

Reply:
fixed :)